src/services/devHandler/newsProcessDevProcess.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable camelcase */
+const {v4: uuidv4} = require('uuid');
+const {newsQueue} = require('../../config');
+const {checkIfValidURL, successResponse, errorResponse} = require('../../utils');
+
+const newsProcessDevProcess = async (req, res) => {
+  const bodyData = req.body?.message;
+  const id_feed = req.body?.id || false;
+  const user_id = req.body?.actor?.id || false;
+  const topics = req.body?.topics || [];
+  const duration_feed = req.body?.duration_feed || '';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining (`?.`) is great here, but you might want to consider providing default values for these variables when they are falsy. This will prevent potential issues later on if these variables are used in a context where their expected type matters.

```diff
  const bodyData = req.body?.message || '';
  const id_feed = req.body?.id || null;
  const user_id = req.body?.actor?.id || null;
  const topics = req.body?.topics || [];
  const duration_feed = req.body?.duration_feed || '';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->